src/components/UserList/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from "react";
+import { useSearchParams } from "react-router-dom";
+import { formatLongDate } from "../../utils/date";
+import { DataTable } from "../DataTable";
+
+type UserListProps = {
+  data: any[];
+  isLoading?: boolean;
+};
+
+const DateCell = ({ cell: { value } }: { cell: { value: Date } }) =>
+  formatLongDate(value);
+
+const columns = [
+  {
+    Header: "Name",
+    accessor: "name"
+  },
+  {
+    Header: "Email",
+    accessor: "email",
+    Aggregated: ""
+  },
+  {
+    Header: "Created At",
+    accessor: "created_at",
+    Cell: DateCell,
+    sortType: "datetime",
+    Aggregated: ""
+  },
+  {
+    Header: "State",
+    accessor: "state",
+    Aggregated: ""
+  }
+];
+
+export function UserList({ data, isLoading }: UserListProps) {
+  const [queryParams, setQueryParams] = useSearchParams({
```

Comment:
Is this not something that should go into DataTable seeing as we do it in many places ?